docs/vm-installation.md

```diff
@@ -0,0 +1,155 @@
+```
+
+4. Create `antrea-agent.kubeconfig` file for `antrea-agent` to access the K8S API server.
+
+```bash
+export CLUSTER_NAME="kubernetes"
+export SERVICE_ACCOUNT="vm-agent"
+APISERVER=$(kubectl config view -o jsonpath="{.clusters[?(@.name==\"$CLUSTER_NAME\")].cluster.server}")
+TOKEN=$(kubectl -n vm-ns get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='$SERVICE_ACCOUNT')].data.token}"|base64 --decode)
+kubectl config --kubeconfig=antrea-agent.kubeconfig set-cluster kubernetes --server=$APISERVER --insecure-skip-tls-verify=true
+kubectl config --kubeconfig=antrea-agent.kubeconfig set-credentials antrea-agent --token=$TOKEN
+kubectl config --kubeconfig=antrea-agent.kubeconfig set-context antrea-agent@kubernetes --cluster=kubernetes --user=antrea-agent
+kubectl config --kubeconfig=antrea-agent.kubeconfig use-context antrea-agent@kubernetes
+# Copy antrea-agent.kubeconfig to VM
+```
+
+5. Create `antrea-agent.antrea.kubeconfig` file for `antrea-agent` to access the `antrea-controller` API server.
+
+```bash
+# Specify the antrea-controller API server endpoint. Antrea-Controller needs to be exposed via the Node IP or a
+# public IP that is reachable from the VM
+export ANTREA_API_SERVER="https://172.18.0.1:443"
+TOKEN=$(kubectl -n vm-ns get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='$SERVICE_ACCOUNT')].data.token}"|base64 --decode)
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig set-cluster antrea --server=$ANTREA_API_SERVER --insecure-skip-tls-verify=true
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig set-credentials antrea-agent --token=$TOKEN
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig set-context antrea-agent@antrea --cluster=antrea --user=antrea-agent
+kubectl config --kubeconfig=antrea-agent.antrea.kubeconfig use-context antrea-agent@antrea
+# Copy antrea-agent.antrea.kubeconfig to VM
+```
+
+## Installation on Linux VM
+
+### Prerequisites
+
+OVS needs to be installed on the VM. For more information about OVS installation please refer to the [getting-started guide](getting-started.md#open-vswitch).
+
+### Installation
+
+1. Build `antrea-agent` binary in the root of the antrea code tree and copy the binary to Linux VM.
+
+```bash
+make docker-bin
+```
+
+2. Copy and update the `antrea-agent` configuration file which specifies agent configuration parameters. Edit
+   antrea-agent.conf file to set `clientConnection`,`antreaClientConnection` and `externalNodeNamespace` with the
+   correct values. Please refer to this [base configuration file](../build/yamls/externalnode/conf/antrea-agent.conf)
+   to check all the configuration parameters for VMs.
+
+```bash
+AGENT_NAMESPACE="vm-ns"
+AGENT_CONF_PATH="/var/run/antrea"
+mkdir -p $AGENT_CONF_PATH
+# Copy antrea-agent kubeconfig files
+cp ./antrea-agent.*kubeconfig $AGENT_CONF_PATH
```

Comment:
I think better to provide the two files to copy, so it is easier for readers to know.

Reply:
Sure